.gitignore

```diff
@@ -1,4 +1,4 @@
-dist/
-grabit
-grabit.lock
-coverage.out
+.DS_Store
+*.bz2
+cmd/test.txt
+cmd/6ae8a75555209fd6c44157c0aed8016e763ff435a19cf186f76863140143ff72
```

Comment:
Please remove the test artifacts from there.  Using a specific hash may not always be correct (e.g., if the file changes someday).  If you're looking to git ignore files that are generated from `make test`, for example, then it might be a good idea to ensure that all those files are in a specific temporary directory, and then ignore that.  But such a change like that should be in a separate PR.